todo_project/db/config.py

```diff
@@ -0,0 +1,24 @@
+from django.conf import settings
+from pymongo import MongoClient
+from todo_project.constants.database import SERVER_SELECTION_TIMEOUT_MS, CONNECT_TIMEOUT_MS
+
+database_client = None
+db = None
+
+
+def get_database_client():
+    global database_client
+    if database_client is None:
+        database_client = MongoClient(
+            settings.MONGODB_URI,
+            serverSelectionTimeoutMS=SERVER_SELECTION_TIMEOUT_MS,
+            connectTimeoutMS=CONNECT_TIMEOUT_MS,
+        )
+    return database_client
+
+
+def get_database():
+    global db
+    if db is None:
+        db = get_database_client()[settings.DB_NAME]
+    return db
```

Comment:
- instead of creating a function that gives us access to database client can we create a class that gives us access to the `ping` and a method to access the collections?
- if you're creating a class make sure we do not create new instance of it everytime the class is called.

Reply:
Done. Created a `DatabaseManager` class exposing the get_collection and `check_database_health` methods